src/main/kotlin/it/pagopa/ecommerce/helpdesk/utils/ResponseBuilderV2.kt

```diff
@@ -221,8 +221,7 @@ fun getBrand(authorizationRequestedData: TransactionGatewayAuthorizationRequeste
         is NpgTransactionGatewayAuthorizationRequestedData -> authorizationRequestedData.brand
         is PgsTransactionGatewayAuthorizationRequestedData ->
             authorizationRequestedData.brand?.toString()
-        is RedirectTransactionGatewayAuthorizationRequestedData ->
-            authorizationRequestedData.paymentMethodType.toString()
+        is RedirectTransactionGatewayAuthorizationRequestedData -> "N/A"
```

Comment:
what about put here the payment type code? 
But maybe it should be a different field in the response 🤔

Reply:
If we do need it I would address it in another PR.
For now we do have the `paymentMethodName` which is enough to get which redirect gateway was chosen